.devcontainer/requirements.txt

```diff
@@ -0,0 +1,14 @@
+.
+./plugins/auth
+./plugins/contents
+./plugins/jupyterlab
+./plugins/jupyterlab
```

Comment:
```suggestion
./plugins/jupyterlab
```

Reply:
Aha thanks, didn't notice that :stuck_out_tongue: